### ISSUE TITLE
Implement Borrow<str> for KeyRef<std::string::String>?

### ISSUE THREAD

**Issue by pourplusquoi** [jeromefroe/lru-rs#117]
Use case:

```
let lru: LruCache::<String, String> = LruCache::new(16);
let _ = lru.get("foo");
```

The compiler complains: the trait bound `KeyRef<std::string::String>: Borrow<str>` is not satisfied.

I'm using `lru = "0.6"`

**Comment by jeromefroe** [comment]
Hi @pourplusquoi! Can you try [version 0.7](https://crates.io/crates/lru/0.7.0) (I just published it)? It includes the changes in #115 that I think should address your issue.

**Comment by pourplusquoi** [comment]
> Hi @pourplusquoi! Can you try [version 0.7](https://crates.io/crates/lru/0.7.0) (I just published it)? It includes the changes in #115 that I think should address your issue.

Hi @jeromefroe, thanks for your reply! It works on version 0.7
